<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Leelawadee UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>900</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>673</width>
      <height>898</height>
     </rect>
    </property>
    <widget class="QPushButton" name="image_btn">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>47</y>
       <width>579</width>
       <height>352</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures/plus-lg 9.png</normaloff>pictures/plus-lg 9.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>93</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="title">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>432</y>
       <width>579</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 34pt &quot;Leelawadee UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(44, 44, 44);</string>
     </property>
     <property name="inputMask">
      <string>Title</string>
     </property>
     <property name="cursorPosition">
      <number>5</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="main_text">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>567</y>
       <width>579</width>
       <height>333</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Leelawadee UI&quot;;
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoBulletList</set>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>507</y>
       <width>87</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Leelawadee UI&quot;;
background-color: rgb(180, 224, 169);
border-radius: 15px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>add</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures/bold plus.png</normaloff>pictures/bold plus.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>0</y>
     <width>16</width>
     <height>900</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ingr_border">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>47</y>
     <width>330</width>
     <height>465</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgb(142, 222, 198);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ingr_lable">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>65</y>
     <width>293</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Leelawadee UI&quot;;
color: rgb(44, 44, 44);</string>
   </property>
   <property name="text">
    <string>Ingridients</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ingridients">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>115</y>
     <width>255</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Leelawadee UI&quot;;</string>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoBulletList</set>
   </property>
  </widget>
  <widget class="QPushButton" name="source_btn">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>596</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(142, 222, 198);
font: 16pt &quot;Leelawadee UI&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>977</x>
     <y>603</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pictures/trash3 1.png</normaloff>pictures/trash3 1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="commit_btn">
   <property name="geometry">
    <rect>
     <x>963</x>
     <y>814</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgb(142, 222, 198);
background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pictures/check2 1.png</normaloff>pictures/check2 1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="cost_line">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>534</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgb(142, 222, 198);
border-radius: 15px;
font: 16pt &quot;Leelawadee UI&quot;;</string>
   </property>
   <property name="inputMask">
    <string>Cost:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
